filters/fadein/fadein.go

```diff
@@ -236,12 +237,12 @@ func (p *postProcessor) Do(r []*routing.Route) []*routing.Route {
 			}
 
 			key := endpointKey(s, h)
-			detected := p.detected[key].when
+			detected := endpointregistry.GetDetectedTime(h)
```

Comment:
We need to pay attention/define what is the endpoint key. Here you use hoist and fadeIn uses scheme+host

Reply:
We get scheme+host from kubernetes endpoints IIRC, that's why we use it for redis, too. Maybe we can do this along with redis.